plugins/dev-tools/src/common_test_helpers.mocha.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * Run configuration information.
+ * @typedef {{
+ *            title:string,
+ *            value:*
+ *            skip:(boolean|undefined),
+ *            only:(boolean|undefined),
+ *            expectedValue:?
+ *          }}
+ */
+let Run;
```

Comment:
`Run` isn't a very specific/descriptive name. I prefer `TestRun` or `TestCase`.

Reply:
I like TestCase, I can do that refactor. A few more files will end up being added, but it should just be JSDoc changes.